index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Apple Old Website</title>
+  <link rel="stylesheet" href="./css/style.css">
+  <script src="https://kit.fontawesome.com/fdaa9ad399.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+  <header>
+    <nav>
+      <ul class="nav-menu">
+        <li class="nav-link nav-logo"><a href="#"><i class="fab fa-apple"></i></a></li>
+        <li class="nav-link nav-link-text"><a href="#">Store</a></li>
+        <li class="nav-link nav-link-text"><a href="#">Mac</a></li>
+        <li class="nav-link nav-link-text"><a href="#">iPod</a></li>
+        <li class="nav-link nav-link-text"><a href="#">iPhone</a></li>
+        <li class="nav-link nav-link-text"><a href="#">iPad</a></li>
+        <li class="nav-link nav-link-text"><a href="#">iTunes</a></li>
+        <li class="nav-link nav-link-text"><a href="#">Support</a></li>
+        <li class="nav-input"><i class="fa fa-search"></i><input type="text"></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <div class="padding-top">
+      <div class="first-image-overflow">
+        <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/your-verse/">
+          <img class="image-text" src="./images/1.png" alt="your-verse">
+        </a>
+      </div>
+      <div>
+        <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/your-verse/">
+          <img class="image-text" src="./images/your_verse_subhead_small_2x.png" alt="everyone has something">
+        </a>
+      </div>
+      <div class="last-text">
+        <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/your-verse/#video-your-verse">
+          <img class="image-text second-image-overflow" src="./images/cta_your_verse_27pt_2x.png" alt="watch the verse">
+        </a>
+      </div>
+    </div>
+  </main>
+
+  <aside>
+    <div><img src="images/3.jpg" alt="icon"></div>
+    <div><img src="images/4.jpg" alt="icon"></div>
+    <div><img src="images/5.jpg" alt="icon"></div>
+    <div><img src="images/6.jpg" alt="icon"></div>
```

Comment:
- [ ] They should be links (as is the original)

![image](https://user-images.githubusercontent.com/61479648/91717252-85f7f480-eb67-11ea-90b6-6193ba262d2a.png)